packages/nimble-components/src/wafer-map/template.ts

```diff
@@ -3,11 +3,13 @@ import type { WaferMap } from '.';
 
 export const template = html<WaferMap>`
     <div class="wafer-map-container">
-        <svg class="svg-root ${x => x.orientation}">
-            <g class="zoom-container">               
-                <svg class="circle-base" version="1.1" x="0px" y="0px" viewbox="1 .45 20 21">
-                    <path class="circle-drawing-path" d="m 21 12 a 10 10 330 1 1 0 -1.98 a 1 1 0 0 0 0 2"/>
-                </svg>
+        <svg class="svg-root">
+            <g class="zoom-container" transform="translate(0,0) scale(1)" ${ref('zoomContainer')}>
```

Comment:
yeah I put that there mostly because of the tests. When I check the transform on a wafer that never had the 'zoom' event fired there is no transform on it and the test checked for undefined which looked a bit off.